src/pages/teams/[teamId]/dashboard/residents/[residentId]/document/[documentSubmissionId].tsx

```diff
@@ -229,10 +229,10 @@ export const getServerSideProps = withAuth(async (ctx) => {
   );
   const resident = await evidenceApiGateway.getResident(user.email, residentId);
 
-  let documentAsBase64 = '';
+  let downloadUrl = '';
   if (documentSubmission && documentSubmission.document) {
```

Comment:
```suggestion
  if (documentSubmission && documentSubmission.claimId) {
```
I think we want to test if the claimId exists instead of checking whether the document exists

Reply:
👍